module/Education/view/education/education/course.phtml

```diff
@@ -6,86 +6,96 @@ $this->headTitle($this->translate('Education'));
-                                        $name = $this->translate('Other exam material from %s (%s)');
-                                        break;
-                                }
-                                ?>
-                                <?= sprintf($name, $exam->getDate()->format('Y-m-d'), $exam->getLanguage()) ?>
-                            <?php endif ?>
-                        </a></li>
-                <?php endforeach; ?>
-            <?php endforeach; ?>
-            <?php foreach ($course->getExams() as $exam): ?>
-                <li><a href="<?= $this->url('education/course/download', [
-                        'code' => $exam->getCourse()->getCode(),
-                        'id' => $exam->getId()
-                    ]) ?>">
-                        <?php if ($exam instanceof Summary): ?>
-                            <?= sprintf($this->translate('Summary by %s on %s (%s)'), $exam->getAuthor(), $exam->getDate()->format('Y-m-d'), $exam->getLanguage()) ?>
-                        <?php else: ?>
-                            <?php
-                            switch ($exam->getExamType()) {
-                                case Exam::EXAM_TYPE_FINAL:
-                                    $name = $this->translate('Examination from %s (%s)');
-                                    break;
-                                case Exam::EXAM_TYPE_INTERMEDIATE_TEST:
-                                    $name = $this->translate('Intermediate test from %s (%s)');
-                                    break;
-                                case Exam::EXAM_TYPE_ANSWERS:
-                                    $name = $this->translate('Answers from %s (%s)');
-                                    break;
-                                case Exam::EXAM_TYPE_OTHER:
-                                    $name = $this->translate('Other exam material from %s (%s)');
-                                    break;
-                            }
-                            ?>
-                            <?= sprintf($name, $exam->getDate()->format('Y-m-d'), $exam->getLanguage()) ?>
-                        <?php endif ?>
-                    </a></li>
-            <?php endforeach; ?>
-        </ul>
+        <div class="row">
+            <div class="col-md-12">
+                <h1><?= $this->escapeHtml($course->getCode()) ?> - <?= $this->escapeHtml($course->getName()) ?></h1>
+            </div>
+            <?php
+            if (null !== $course->getUrl()):
+                ?>
+                <div class="col-md-12">
+                    <a href="<?= $course->getUrl() ?>"><?= $this->escapeHtml($course->getUrl()) ?></a>
+                </div>
+            <?php
+            endif;
+            ?>
+        </div>
+        <hr>
+        <div class="row">
+            <div class="col-md-4">
+                <h2><?= $this->translate('Subcourses') ?></h2>
+                <?php if (!$course->getChildren()->isEmpty()): ?>
```

Comment:
I would put this if statement before opening `<div class="col-md-4">`. Otherwise the page still looks a bit odd if there are no subcourses. Then you could also completely leave out the else statement.

Reply:
Yeah I guess that for the subcourses that is preferable.